Integration/AndroidAAR/BabylonNative/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.13.2)
+# set targetPlatform, will be passed in from gradle when this sample is completed
+# openGL Supportability
+# platform         status
+#   (0 12)           ES2/ES3 not supported
+#   [12, 18)         ES2 only; for ES3, app do dynamic load/detection
+#                    this applies to the situations that:
+#                        - minimum API is set to less than 18. In this case
+#                          there is no ES3 header/lib support inside NDK
+#                        - the built APK might be running on newer API phones
+#                    with dynamic loading of ES3, the same APK would still be able
+#                    to use ES3. Otherwise, app would stuck with ES2 even phone is
+#                    is newer than the minimum API level (for example, Android-27 etc).
+#
+#   [18, 24)         ES2 & ES3
+#                    If app is built to only support API-18 or later,
+#                    set minimum api level to 18 is good enough, NDK supprts ES3
+#                    with the right header and lib files. No need to use ES3 dynamic
+#                    detection.
+#   [24, infinite)   ES2 & ES3 & Vulkan
+project(BabylonNative)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+set(BABYLON_NATIVE_PLATFORM "Android")
+set(OPENGL_LIB GLESv3)
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/src/")
+
+set(BABYLON_NATIVE_PACKAGE_DIR "${CMAKE_CURRENT_LIST_DIR}/..")
+add_subdirectory(${BABYLON_NATIVE_PACKAGE_DIR}/../../Library/ ${BABYLON_NATIVE_PACKAGE_DIR}/../../build/Android_${CMAKE_ANDROID_ARCH_ABI}/)
+
+add_library(BabylonNativeJNI SHARED
+            src/main/cpp/BabylonNativeJNI.cpp
+            ${BABYLON_NATIVE_PACKAGE_DIR}/../../Library/Dependencies/napi/source/js_native_api_v8.cc
```

Comment:
v8 is not supported in cmake for Android. This needs more work to pull the npm, untar,... There is an issue : https://github.com/BabylonJS/BabylonNative/issues/102#issuecomment-554454182

Reply:
It's more complicated than it looks (see the issue linked). I think it can be done with less work on the CI because it's using a Mac image to build Android. On Windows, it needs tools to uncompress the archive.

Reply:
Should we add a comment here with a link to the issue tracking this?

Reply:
Comment added.